addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15782,32 +15792,32 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36410"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36411"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36412"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36413"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36414"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36415"
-msgid "No info available yet."
+msgid "No information available yet."
```

Comment:
sorry to probably open pandoras box, but instead of changing these strings we should actually write a proper description instead. These strings are only placeholders that should have never made it into a release. Will see if I can come up with suggestions for these.